src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -90,41 +221,54 @@
           "type": "string"
         }
       },
+      "required": [
+        "scheduler"
+      ],
       "type": "object"
     },
     "preprocessing": {
+      "additionalProperties": false,
       "properties": {
         "lateral_boundary_conditions": {
+          "additionalProperties": false,
           "properties": {
             "interval_hours": {
-              "default": 3,
               "minimum": 1,
-              "type": "number"
+              "type": "integer"
             },
             "offset": {
-              "default": 0,
               "minimum": 0,
-              "type": "number"
+              "type": "integer"
             },
             "output_file_path": {
-              "format": "uri",
               "type": "string"
             }
           },
+          "required": [
+            "interval_hours",
+            "offset",
+            "output_file_path"
+          ],
           "type": "object"
         }
       },
+      "required": [
+        "lateral_boundary_conditions"
+      ],
       "type": "object"
     },
     "user": {
+      "additionalProperties": false,
```

Comment:
I think we will want the `user` section to be able to set anything. That's where they might do anything they like, such as tossing in re-usable variables that can be referenced in a `realize` step.

Reply:
Having a place for free-form user config sounds like a good idea, but I don't think that should be mixed with schema-controlled content. Should `account` perhaps move next to `scheduler`, as they are both related to the batch system?

Reply:
I like that idea. Let's move it.

Reply:
Moved `account` to `platform:` block alongside `mpicmd` and `scheduler`. Made it required iff `scheduler` is defined.